UnityProject/Assets/Scripts/Systems/Inventory/Inventory.cs

```diff
@@ -656,15 +656,25 @@ public static void PopulateSubInventory(ItemStorage itemStorage, List<SlotPopula
 
 		foreach (var namedSlotPopulatorEntry in namedSlotPopulatorEntrys)
 		{
+			if (namedSlotPopulatorEntry == null || namedSlotPopulatorEntry.Prefab == null) continue;
 			ItemSlot ItemSlot;
-			if (namedSlotPopulatorEntry.UesIndex)
+			if (namedSlotPopulatorEntry.DoNotGetFirstEmptySlot == false)
 			{
-				ItemSlot = itemStorage.GetIndexedItemSlot(namedSlotPopulatorEntry.IndexSlot);
+				ItemSlot =  itemStorage.GetNextEmptySlot();
 			}
 			else
 			{
-				ItemSlot = itemStorage.GetNamedItemSlot(namedSlotPopulatorEntry.NamedSlot);
+				if (namedSlotPopulatorEntry.UesIndex)
```

Comment:
what the hell is a ``UesIndex``

Reply:
no comments? no explanations for the why?

![image](https://user-images.githubusercontent.com/34368774/216483713-812fdbe9-b45c-4d90-ae63-8574b1d9b995.png)